forc-plugins/forc-client/src/cmd/deploy.rs

```diff
@@ -81,4 +81,8 @@ pub struct Command {
     /// Disable the "new encoding" feature
     #[clap(long)]
     pub no_encoding_v1: bool,
+
+    /// Maximum contrat size, after which forc-deploy automatically splits the contract into chunks.
+    #[clap(long)]
+    pub maximum_contract_size: Option<usize>,
```

Comment:
Any reason why we would want to make this configurable rather than just using the `MAX_CONTRACT_SIZE` const?

Reply:
If there isn't a valid reason I think we should remove it

Reply:
Sure, I made it like this as for different networks targeted the size limit for the contract might be different. But I don't think this would be a hugely useful as I am imagining we will have a set contract size limit that won't change in near future. Currently I am using it for testing but i can work around it no problems. Maybe we should look for a way to query the size limit from node. That way the value can be adjusted according to the target :)